Header header
int16 NumTags
int16[] Id
string[] Name
float32[] PosX
float32[] PosY
float32[] PosZ
float32[] SmoPosX
float32[] SmoPosY
float32[] SmoPosZ

